sky/services/media/media.mojom

```diff
@@ -14,7 +14,23 @@ interface MediaPlayer {
   SeekTo(uint32 msec);
 };
 
+[ServiceName="media::SoundPool"]
+interface SoundPool {
+  Load(handle<data_pipe_consumer> data_source) => (bool success, int32 sound_id);
+  Play(int32 sound_id, float left_volume, float right_volume, bool loop, float rate) =>
+      (bool success, int32 stream_id);
+  Stop(int32 stream_id);
+  Pause(int32 stream_id);
+  Resume(int32 stream_id);
+  SetRate(int32 stream_id, float rate);
+  SetVolume(int32 stream_id, float left_volume, float right_volume);
```

Comment:
This API presumably is intended to be backed by more than just Android, though, no?

I'm just saying we should make our API extensible. We don't want to end up with lots of APIs that have weird sendimentary layers right from the start. :-)